src/commands/BrighterScriptPreviewCommand.ts

```diff
@@ -243,7 +243,9 @@ export class BrighterScriptPreviewCommand {
      * Get the fsPath from the uri. this handles both `file` and `bs-preview` schemes
      */
     private getSourcePathFromPreviewUri(uri: vscode.Uri) {
-        if (uri.scheme === FILE_SCHEME) {
+        if (uri.scheme === 'file') {
```

Comment:
This isn't used by RDB but it does surface in the webview tools as warnings which was causing a lot of unnecessary noise. I looked through the old code for this line and added it back in as you had it before to silence it. If there's a better way to go about it happy for you to address that way. I just wanted to avoid all the warnings or errors can't remember which